src/components/controls/styles.js

```diff
@@ -24,19 +24,23 @@ export const ControlsContainer = styled.div`
 export const HeaderContainer = styled.div`
   display: flex;
   justify-content: space-between;
-  font-family: ${(props) => props.theme["font-family"]};
-  font-size: 16px;
   line-height: 28px;
   min-height: 28px; /* needed for safari, else the div height is 0 */
   margin-top: 15px;
   margin-bottom: 5px;
+  border-top: 1px solid #d3d3d3;
```

Comment:
I'd suggest the border is only used to demarcate panels, not everything which uses `<AnnotatedHeader>`. (The sidebar element below the last panel may also want to use this to indicate it's not part of the panel options.)

Reply:
Good suggestion. Updated in 66f82bdb901b37e7875278b4c4a00e251cbdf92b.